### PR TITLE
Display info of 5 books

### PR DESCRIPTION
**Pull request by Serge-prospect**
Please review HW05 - BooksInfo.

### CODE REVIEW

File: BooksInfo/BooksInfo/Program.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {        
+        static void Main(string[] args)
+        {
+            Book[] library = new Book[5];
+
+            // Generate unique book id
+            for (int index = 0; index < library.Length; index++)
+            {
+                string id = (index + 1).ToString();
+                library[index] = new Book(id);
```
Comment (EgorAntonovich) [inline BooksInfo/BooksInfo/Program.cs]: you can set id to `new Book()` like this new Book((index + 1).To String()) if you want.  but better create custom objects for fill array and set info in each property without default values.

File: BooksInfo/BooksInfo/Book.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        public Book(string id)
+        {
+            Id = id;
+            Title = "No title";
+            Pages = 0;            
+        }
+
+        public void DisplayBookInfo()
+        {
+            Console.WriteLine($"ID: {Id}     Title: {Title}     Pages: {Pages}");
```
Comment (EgorAntonovich) [inline BooksInfo/BooksInfo/Book.cs]: you can use \t instead of million spaces

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: BooksInfo/BooksInfo/Program.cs (hunk 1 above)
Comment (Serge-prospect) [reply BooksInfo/BooksInfo/Program.cs]: If I simply add objects with **id**, **title**, **pages**, how can I make sure that **id** value is unique? When I create a **Book** object with **id** (no set) field only in it via adding +1 in **id** in cycle, it is realy unique. Then I fill in other fields of the created **Book** objects by any data as I want.

Comment (Serge-prospect) [reply BooksInfo/BooksInfo/Program.cs]: Changed to another way of getting a unique ID.

**Review by EgorAntonovich — APPROVED** [review]

**Pull request merged**